workers-site/index.js

```diff
@@ -53,17 +53,19 @@ async function handleEvent(event) {
         bypassCache: true,
       }
     }
-    return hydrateEdgeState({
-      response: getAssetFromKV(event, options),
-      state: transformBookmark(event.request),
-    })
+    const response = await getAssetFromKV(event, options)
+    const state = await transformBookmark(event.request)
```

Comment:
The `await` calls need to happen here so that they're caught by this `try`/`catch` block!